src/checks_util.cc

```diff
@@ -32,14 +33,20 @@
 #include "zetasql/public/parse_helpers.h"
 #include "zetasql/public/parse_location.h"
 #include "zetasql/public/parse_resume_location.h"
+#include "zetasql/public/parse_tokens.h"
 
 namespace zetasql::linter {
 
+absl::string_view GetName(const ParseLocationRange &range,
```

Comment:
Since you can use token.GetImage() for tokens, you can change this function to `absl::string_view GetName(const ASTNode& node)`

Reply:
There is already GetNodeString for that

Reply:
Ahm, does it mean that you don't need this function anymore? :) If it is not used anywhere - just inline it into GetNodeString

Reply:
Yes, I erased "GetName"